### PR TITLE
Comment flatlist WIP

### PR DESCRIPTION
**Pull request by curioussavage**
## ticket
https://reddit.atlassian.net/browse/CONT-35

## dependencies
 https://github.com/reddit/node-api-client/pull/197

## staging
https://comment-flatlist.mobile.staging.snooguts.net/

### CODE REVIEW

**Comment by curioussavage** [comment]
👓  @schwers @nramadas

**Comment by curioussavage** [comment]
also 👓  @scarow  because I had to fix some code/tests from mod tools stuff.

**Comment by curioussavage** [comment]
:haircut_man:

**Comment by scarow** [comment]
modtools stuff looks good on staging 👍  thanks for tagging me

File: src/app/components/Comment/index.jsx [hunk 1]
```diff
@@ -1,90 +1,146 @@
...
+      authorType,
+      preview,
+      isTopLevel,
+      isUserActivityPage,
+      highlightedComment,
+      onToggleCollapse,
+      editing,
+      dotsNum,
+    } = props;
+
```
Comment (schwers) [inline src/app/components/Comment/index.jsx]: :nit: extra space

**Comment by curioussavage** [comment]
for informational purposes - https://reddit.atlassian.net/browse/CONT-99 is the jira ticket that covered profiling performance

**Pull request merged**